Resources/Prototypes/Loadouts/Jobs/Security/traitRestrictedGear.yml

```diff
@@ -0,0 +1,635 @@
+  category: JobsSecurityAUncategorized
+  cost: 7
+  exclusive: true
+  requirements:
+    - !type:CharacterTraitRequirement
+      traits:
+        - RestrictedGear
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutSecurityOuter
+    - !type:CharacterDepartmentRequirement
+      inverted: true
+      departments:
+        - Security
+  items:
+    - ClothingOuterArmorPlateCarrier
+
+- type: loadout
+  id: LoadoutSecurityRestrictedGearOuterArmorDuraVest
+  category: JobsSecurityAUncategorized
+  cost: 7
+  exclusive: true
+  requirements:
+    - !type:CharacterTraitRequirement
+      traits:
+        - RestrictedGear
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutSecurityOuter
+    - !type:CharacterDepartmentRequirement
+      inverted: true
+      departments:
+        - Security
+  items:
+    - ClothingOuterArmorDuraVest
+
+- type: loadout
+  id: LoadoutSecurityRestrictedGearOuterArmorBasic
+  category: JobsSecurityAUncategorized
+  cost: 7
+  exclusive: true
+  requirements:
+    - !type:CharacterTraitRequirement
+      traits:
+        - RestrictedGear
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutSecurityOuter
+    - !type:CharacterDepartmentRequirement
+      inverted: true
+      departments:
+        - Security
+  items:
+    - ClothingOuterArmorBasic
+
+- type: loadout
+  id: LoadoutSecurityRestrictedGearOuterArmorSlim
+  category: JobsSecurityAUncategorized
+  cost: 7
+  exclusive: true
+  requirements:
+    - !type:CharacterTraitRequirement
+      traits:
+        - RestrictedGear
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutSecurityOuter
+    - !type:CharacterDepartmentRequirement
+      inverted: true
+      departments:
+        - Security
+  items:
+    - ClothingOuterArmorBasicSlim
+
+
+## Equipment
+# Flash
+- type: loadout
+  id: LoadoutSecurityRestrictedGearFlash
+  category: JobsSecurityAUncategorized
+  cost: 2
+  requirements:
+    - !type:CharacterTraitRequirement
+      traits:
+        - RestrictedGear
+    - !type:CharacterDepartmentTimeRequirement
+      department: Security
+      min: 3600 # 1 hours
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutSecurityEquipment
+    - !type:CharacterDepartmentRequirement
+      inverted: true
+      departments:
+        - Security
+  items:
+    - Flash
+
+# .35 Magazine
+- type: loadout
+  id: LoadoutSecurityRestrictedGearMagazinePistol
+  category: JobsSecurityAUncategorized
+  cost: 2
+  requirements:
+    - !type:CharacterTraitRequirement
+      traits:
+        - RestrictedGear
+    - !type:CharacterDepartmentTimeRequirement
+      department: Security
+      min: 3600 # 1 hours
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutSecurityEquipment
+    - !type:CharacterDepartmentRequirement
+      inverted: true
+      departments:
+        - Security
+  items:
+    - MagazinePistol
+
+# .35 Magazine Rubber
+- type: loadout
```

Comment:
IMO this is the only thing actually contentious about this PR, the rest I'm actually a really huge fan of for this trait, and love the implementation. April says she already has a working implementation of CVarRequirement, so this "RestrictedGearAllowsFirearms" is an obvious good use-case of a CharacterCVarRequirement.

Reply:
ah that'll resolve everything neatly when its ready